src/formio/components/AddressNL.js

```diff
@@ -323,69 +326,78 @@ const FormikAddress = ({required, setFormioValues, deriveAddress}) => {
     setFieldValue('secretStreetCity', data['secretStreetCity']);
   };
 
+  const wrappedWithColumns = fields => {
+    const wrappedFields = fields.map(field => {
+      if (layout === 'doubleColumn') {
+        return <div className="column column--span-6 openforms-form-field-container">{field}</div>;
+      }
+
+      return field;
+    });
+    return layout === 'doubleColumn' ? (
+      <div className="openforms-columns">{wrappedFields}</div>
+    ) : (
+      wrappedFields
+    );
+  };
```

Comment:
I'm wondering if we can clean this code up a bit by defining helper components instead of inlining everything.

Having something like this would be nice I suppose:

```jsx
const useColumns = layout === 'doubleColumn';

<div className="openforms-form-field-container">
  <Row useColumns={useColumns}>
    <Field useColumns={useColumns}>
      <PostCodeField required={required} autoFillAddress={autofillAddress} />
    </Field>
    <Field useColumns={useColumns}>
      <HouseNumberField required={required} autoFillAddress={autofillAddress} />
    </Field>
  </Row>
</div>
```

etc.

I find this reads much easier and you can manage the exact markup in the (new) `Row` and `Field` components.

I'd also instead of doing columns consider using a flat structure and do the layout with CSS grid, so you can toggle it into single/double column mode:

```css
.openforms-addressnl {
  display: grid;
  gap: 20px; // should be design token of course
}

.openforms-addressnl.openforms-addressnl--single-column {
  grid-template-columns: 1fr;  // make it single column
}

.openforms-addressnl.openforms-addressnl--double-column {
  grid-template-columns: 1fr 1fr; // make it double column
}
```

this has the benefit that the markup is much nicer